entc/integration/edgefield/ent/schema/user.go

```diff
@@ -36,6 +36,7 @@ func (User) Fields() []ent.Field {
 func (User) Edges() []ent.Edge {
 	return []ent.Edge{
 		edge.To("pets", Pet.Type),
+		edge.To("previous_pets", Pet.Type).StorageKey(edge.Column("previous_owner_id")),
```

Comment:
Why is StorageKey defined here? It should be defined on the schema that holds the foreign key using `Field` instead. Ent should throw an error in this case. 

Do you have any other case that reproduces it?